packages/ses/docs/lockdown.md

```diff
@@ -166,17 +196,34 @@ lockdown({
   overrideTaming: 'min', // Until https://github.com/endojs/endo/issues/636
 });
 ```
+
+If `lockdown` receives no election for `consoleTaming`, it will respect
+`process.env.LOCKDOWN_CONSOLE_TAMING`.
+
+```console
+LOCKDOWN_CONSOLE_TAMING=safe
+LOCKDOWN_CONSOLE_TAMING=unsafe
+```
+
+The `consoleTaming: 'safe'` setting replaces the global console with a tamed
+console, and that tamed console is safe to endow to a guest `Compartment`.
+Additionally, any errors created with the `assert` function or methods on its
+namespace may have redacted details: information included in the error message
+that is informative to a debugger and made invisible to an attacker.
+The tamed console removes redactions and shows these details to the original
+console.
+
 The `consoleTaming: 'unsafe'` setting leaves the original console in place.
 The `assert` package and error objects will continue to work, but the `console`
 logging output will not show any of this extra information.
 
 The risk is that the original platform-provided `console` object often has
-additional methods beyond the de facto `console` "standards". Under the
-`'unsafe'` setting we do not remove them.
-We do not know whether any of these additional
-methods violate ocap security. Until we know otherwise, we should assume these
-are unsafe. Such a raw `console` object should only be handled by very
-trustworthy code.
+additional methods beyond the de facto `console` "standards" and may be unsafe
+to endow to a guest `Compartment`.
+Under the `'unsafe'` setting we do not remove them.
+We do not know whether any of these additional methods violate ocap security.
+Until we know otherwise, we should assume these are unsafe. Such a raw
+`console` object should only be handled by very trustworthy code.
```

Comment:
FYI https://github.com/endojs/endo/blob/193e40340d053118fd1eeb627c1ed273e27877ea/packages/ses/src/lockdown.js#L312-L314 from @mhofman 's https://github.com/endojs/endo/pull/1939 does mitigate the case we're aware of.